integration_tests/test_dict_06.py

```diff
@@ -0,0 +1,24 @@
+from lpython import f64, i32
+
+def fill_rollnumber2cpi(size: i32) -> dict[i32, f64]:
+    i : i32
+    rollnumber2cpi: dict[i32, f64] = {}
+    
+    rollnumber2cpi[0] = 1.1
+    for i in range(1000, 1000 + size):
+        rollnumber2cpi[i] = float(i/100.0 + 5.0)
+
+    return rollnumber2cpi
+
+def test_dict():
+    i: i32
+    size: i32 = 1000
```

Comment:
```suggestion
    size: i32 = 200
```

Smaller value for quickly running the tests in CPython. `1000` is already used in other test of `dict` so they will ensure proper testing already.

Reply:
I did the changes.

Reply:
@czgdp1807 @certik review.